product_security/red_hat_responsibilities/compliance.adoc

```diff
@@ -0,0 +1,21 @@
+[[dedicated-compliance]]
+= Compliance
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+Compliance of Red Hat {product-title} on AWS against compliance standards such
+as SOC, ISO, PCI DSS, and HIPAA is in progress. As a hosted service provider,
+Red Hat has strong processes aligned with NIST 800-123 standards to manage
+information security. Red Hat implements security controls that ascertain
```

Comment:
I am not sure we should use the language "ascertain" and "ensuring" as those may be legally binding. Check with legal first.

Reply:
@patrickstrick Okay, thanks! I will try to track someone down.

Reply:
@patrickstrick @anjalitelang I opened a ticket with Legal and copied you. The rest of your feedback is now addressed. Thanks!